WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartMySitePrompts.kt

```diff
@@ -13,6 +13,13 @@ enum class QuickStartMySitePrompts constructor(
     val shortMessagePrompt: Int,
     val iconId: Int
 ) {
+    UPDATE_SITE_TITLE_TUTORIAL(
+            QuickStartTask.UPDATE_SITE_TITLE,
+            R.id.my_site_scroll_view_root,
+            R.id.my_site_title_label,
+            R.string.quick_start_dialog_update_site_title_message_short,
+            -1
```

Comment:
How would you feel about extracting the `-1` into a constant? Something like `ICON_NOT_SET`

Reply:
Sounds good to me 👍